R/dataframe.R

```diff
@@ -229,6 +229,12 @@ as_tibble.default <- function(x, ...) {
 
 #' @export
 #' @rdname as_tibble
+as.tibble <- function(x, ...) {
```

Comment:
Your way creates a new function, the existing alias is just that. The differences are subtle and perhaps unimportant, but we should strive for consistency here. I'm surprised that `UseMethod("as_tibble")` works here -- I'd have thought it should read `UseMethod("as_data_frame")`.